Source/Core/DolphinTool/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 add_executable(dolphin-tool
   ToolHeadlessPlatform.cpp
-  Command.h
```

Comment:
Please also remove this from the VS files, it's in both `DolphinTool.vcxproj` and `DolphinTool.filters`.

Reply:
Oops.  Wait, how was this compiling at all??  HeaderCommand was never put in there.